docs/rules/computed-property-spacing.md

```diff
@@ -14,6 +14,9 @@ var a = "prop";
 var obj = {
   [a]: "value" // computed property key in object literal (ECMAScript 6)
 };
+
+// applies to the spacing for dynamic keys (computed property key) when destructuring objects.
+const ({ [a]: someProp } = obj); // someProp now maps to the value of prop.
```

Comment:
Parentheses are invalid syntax here:

```suggestion
const { [a]: someProp } = obj; // someProp now maps to the value of prop.
```

Reply:
Wait, I think I did not read this right, that syntax looks wrong, that works, when assigning to a delcared variable:

```js
let someProp;
({ [a]: someProp } = obj);
```

but if we are declaring a new variable:

```js
const { [a]: someProp } = obj;
```

this is mentioned in this section: [Assignment without declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)